lib/auth.coffee

```diff
@@ -215,6 +215,41 @@ getAuth = (deps, opts) ->
 		.catchReturn(false)
 		.asCallback(callback)
 
+	###*
+	# @summary Generate a new API key for user
+	# @name generateApiKey
```

Comment:
`generate` slightly suggests that this method doesn't have side effects, it just locally generates a key somehow, which isn't quite right.

Can we go for `createApiKey` instead? That's a little clearer, and also matches the `(device/application/etc).create` methods in the rest of the SDK.